packages/server/src/accounts-server.ts

```diff
@@ -571,6 +574,13 @@ Please change it with a strong random token.`);
       : generateRandomToken();
   }
 
+  private async createJwtPayload(data: JwtData, user: User): Promise<JwtPayload> {
+    if (this.options.jwtPayloadCreator) {
+      return await this.options.jwtPayloadCreator.createPayload(data, user);
```

Comment:
does it make sense to allow `jwtPayloadCreator.createPayload` to modify data?
I think doing something like 
```
return {
...await this.options.jwtPayloadCreator.createPayload(data, user)
 data
}
```
would be safer.